libs/orders/src/lib/components/order-feedback/order-feedback.tsx

```diff
@@ -113,6 +74,27 @@ export const OrderFeedback = ({ transaction, order }: OrderFeedbackProps) => {
           </div>
         )}
       </div>
+      {orderRejectionReason && (
+        <div>
+          <p className={labelClass}>{t(`Reason`)}</p>
+          <p data-testid="error-reason">{t(orderRejectionReason)}</p>
+        </div>
+      )}
     </div>
   );
 };
+
+const rejectionReason = (
```

Comment:
nit (non-blocking): i find it a lot more readable when looking at a variable and being able to assume it's a function. usually prefixing it with a verb makes it obvious, like `getRejectionReason`. no big deal tho, pls free free to resolve this without action

Reply:
updated